<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetAirconditioner</class>
 <widget class="QWidget" name="SetAirconditioner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: rgb(51,153,255);
}

QLabel
{
background-color:rgb(51,153,255);
border:0px;
font:20px &quot;SimHei&quot;;
color:black;
}

QPushButton
{
outline:none;
border-image: url(:/image/buttonUP.jpg);
font:20px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;
}

QPushButton:disabled
{
outline:none;
border:1px solid white;
border-image: url(:/image/buttonUP.jpg);
font:25px, &quot;SimHei&quot;;
color:rgb(180, 180, 180);
border-radius:2px;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(191,191,191);
border:1px solid black;</string>
   </property>
   <property name="text">
    <string>显示项目</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(191,191,191);
border:1px solid black;</string>
   </property>
   <property name="text">
    <string>C1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(191,191,191);
border:1px solid black;</string>
   </property>
   <property name="text">
    <string>C2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelCompressorC1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border:1px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelCompressorC2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>40</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border:1px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelCompressor">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border:1px solid black;</string>
   </property>
   <property name="text">
    <string>压缩机</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelVentilateC1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border:1px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelVentilate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border:1px solid black;</string>
   </property>
   <property name="text">
    <string>通风机</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelVentilateC2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>70</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border:1px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelCondenserC1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border:1px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelCondenser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border:1px solid black;</string>
   </property>
   <property name="text">
    <string>冷凝风机</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelCondenserC2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>100</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border:1px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelHeater1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border:1px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelNewWind">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border:1px solid black;</string>
   </property>
   <property name="text">
    <string>新风温度℃</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelNewWind2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>160</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border:1px solid black;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelReturnWind1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border:1px solid black;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelReturnWind2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>190</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border:1px solid black;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelReturnWind">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border:1px solid black;</string>
   </property>
   <property name="text">
    <string>回风温度℃</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelHeater">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border:1px solid black;</string>
   </property>
   <property name="text">
    <string>电热器</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelHeater2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>130</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border:1px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelNewWind1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border:1px solid black;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelWorkMode">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border:1px solid black;</string>
   </property>
   <property name="text">
    <string>工作模式</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelWorkMode2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>220</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border:1px solid black;</string>
   </property>
   <property name="text">
    <string>关机</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelWorkMode1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border:1px solid black;</string>
   </property>
   <property name="text">
    <string>关机</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnONOff">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>90</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>空调关</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPreOff">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>305</y>
     <width>90</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>预冷/暖关</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAutoCold">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>90</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>自动冷</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAutoWarm">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>305</y>
     <width>90</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>自动暖</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnManuCold">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>90</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>手动冷</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnManuWarm">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>305</y>
     <width>90</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>手动暖</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnWentilate">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>90</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>通风</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSetTem">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>280</y>
     <width>90</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>温度设定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>280</y>
     <width>90</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTem">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>285</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border:1px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>255</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMinus">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>305</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCompressor11">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelCompressor12">
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>40</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelCompressor21">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>40</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelCompressor22">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>40</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelVentilate1">
   <property name="geometry">
    <rect>
     <x>305</x>
     <y>70</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelCondenser1">
   <property name="geometry">
    <rect>
     <x>305</x>
     <y>100</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelHeater11">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelCondenser2">
   <property name="geometry">
    <rect>
     <x>605</x>
     <y>100</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelVentilate2">
   <property name="geometry">
    <rect>
     <x>605</x>
     <y>70</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelHeater21">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>130</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelModeInfo">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>320</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelHeater12">
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>130</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelHeater22">
   <property name="geometry">
    <rect>
     <x>685</x>
     <y>130</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
